src/x11_in_gtk2.c

```diff
@@ -83,7 +83,9 @@ x_window_is_valid(SuilX11Wrapper* socket)
 			return true;
 		}
 	}
-	XFree(children);
+	if (children) {
```

Comment:
Why?   XFree(3): "If data is NULL, no operation is performed."

Reply:
https://tronche.com/gui/x/xlib/display/XFree.html
`A NULL pointer cannot be passed to this function.`

Maybe xlib vs. xcb?

Reply:
Hm.  Weird.